mirascope/base/utils.py

```diff
@@ -265,3 +268,45 @@ async def wrapper_generator_async(*args, **kwargs):
         return cast(F, wrapper_generator_async)
     else:
         return cast(F, wrapper)
+
+
+def get_class_vars(cls: type[BaseModel]) -> dict[str, Any]:
+    """Get the class variables of a `BaseModel` removing any dangerous variables."""
+    class_vars = {}
+    for classvars in cls.__class_vars__:
+        if not classvars == "api_key":
+            class_vars[classvars] = getattr(cls.__class__, classvars)
+    return class_vars
+
+
+def get_wrapped_call(call: Callable, cls: "BaseCall", **kwargs) -> Callable:
```

Comment:
note: you can make this just `BaseCall` if you also import `from __future__ import annotations`